lib/kernel.ts

```diff
@@ -457,7 +459,7 @@ export class Kernel {
 		]);
 
 		const adminUser = await unsafeUpsert(CONTRACTS['user-admin']);
-		const adminSession = await unsafeUpsert({
+		await unsafeUpsert({
```

Comment:
shouldn't this be removed entirely?

Reply:
@Ereski Nice catch, removed it and confirmed that everything passes. Rebased and pushed the changes.